src/Smithers.js

```diff
@@ -22,4 +22,34 @@ export default class Smithers {
         const requestUrl = createPath(paths.lastBuild, { name });
         return this.caller.get(requestUrl, config);
     }
+
+    lastSuccessfulBuild(name, config) {
+        const requestUrl = createPath(paths.lastSuccessfulBuild, { name });
+        return this.caller.get(requestUrl, config);
+    }
+
+    lastStableBuild(name, config) {
+        const requestUrl = createPath(paths.lastStableBuild, { name });
+        return this.caller.get(requestUrl, config);
+    }
+
+    lastUnsuccessfulBuild(name, config) {
+        const requestUrl = createPath(paths.lastUnsuccessfulBuild, { name });
+        return this.caller.get(requestUrl, config);
+    }
+
+    specificBuild(name, build, config) {
```

Comment:
`build` looks better as `buildNumber` like you've put in the README

Reply:
For me `buildNumber` looks more verbose here

Reply:
Sorry, could have been clearer!